homework_week1/homework_week1/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  background-color: bisque;
+}
+
+h1 {
+  text-align: center;
+  font-style: italic;
+}
+h4 {
+  color: brown;
+  font-style: oblique;
+}
+
+#pride_and_prejudice img {
+  content: url('https://images.penguinrandomhouse.com/cover/9780307950901');
```

Comment:
If you have fixed the js to display the images you don't need them here anymore.